docs/admin/config.rst

```diff
@@ -725,6 +725,63 @@ GET_HELP_URL
 
 URL where support for your Weblate instance can be found.
 
+.. setting:: GITEA_CREDENTIALS
+
+GITEA_CREDENTIALS
+------------------
+
+.. versionadded:: 4.3
```

Comment:
This is just copied right now, what should I add as the next version number?

Reply:
Please use 4.12 here.